LICENSE

```diff
@@ -428,13 +428,12 @@ Copyright 2021- HPC-AI Technology Inc. All rights reserved.
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
-   ---------------- LICENSE FOR AutoGPTQ ----------------
 
-   From AutoGPTQ:
+   ---------------- LICENSE FOR LangChain TEAM ----------------
 
-   MIT License
+   The MIT License
 
-   Copyright (c) 2023 潘其威(William)
+   Copyright (c) Harrison Chase
```

Comment:
You should add license rather than modifying

Reply:
sorry this pr is outdated, will fix and push again